oneanddone/tasks/migrations/0022_make_taskname_unique.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+from south.v2 import DataMigration
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        changed = set()
+        for task in orm.Task.objects.all():
+            value = getattr(task, 'name')
+            duplicates = orm.Task.objects.filter(**{'name': value}).exclude(pk=task.pk)
+            for i, dupe in enumerate(duplicates):
+                if dupe.pk in changed:
+                    continue
+                changed.add(task.pk)
+                changed.add(dupe.pk)
+                setattr(dupe, 'name', value + str(i+2))
```

Comment:
I would suggest adding a space before the number that you are appending as it may make the title a bit more readable.